brave/src/main/java/com/linecorp/armeria/client/brave/ClientRequestContextAdapter.java

```diff
@@ -89,7 +89,9 @@ public String header(String name) {
 
         @Override
         public void header(String name, String value) {
-            headersBuilder.set(name, value);
+            if (headersBuilder != null) {
```

Comment:
when serving the request from the response side. I'm thinking we ignore header mutations..

Reply:
we could before, just htis will make that more explicit https://github.com/openzipkin/brave/pull/1103